docs/DeveloperGuide.md

```diff
@@ -170,6 +170,35 @@ sequence diagram shows the interactions within `Model` when editing a person.
 
 ![EditSequenceDiagram-Model](images/EditSequenceDiagram-Model.png)
 
+### Meeting Scheduling
+
+`Person` class has an `Optional<Meeting>` field which will hold a `Meeting` object that contains the meeting details if a meeting is scheduled with the person. Otherwise, it will hold an `Optional.empty()` to represent no meeting is scheduled with the person.
+
+The supported meeting details are:
+- Start date time
+- End date time
+- Remarks
+
+#### Schedule Command
+
+The `schedule` command is implemented to allow users to schedule meetings within the application. The command follows a sequence of interactions similar to the other commands. The part to highlight is `ScheduleCommandParser#parse(String)`  creates a `Meeting` object containing the parsed meeting details and it is then passed to `ScheduleCommand`.
+
+The following sequence diagram shows how a schedule meeting operation goes through the `Logic` component:
+![Schedule Meeting Sequence Diagram](images/ScheduleMeetingSequenceDiagram.png)
+
+<div markdown="span" class="alert alert-info">:information_source: **Note:** The lifeline for `ScheduleCommandParser` and `ScheduleCommand` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline continues till the end of diagram.
+</div>
```

Comment:
LGTM, perhaps a suggetion for the team is that we can move this up to the top when finalising our DG since this will apply to all sequence diagrams